rotkehlchen/rotkehlchen.py

```diff
@@ -256,10 +258,20 @@ def unlock_user(
             greenlet_manager=self.greenlet_manager,
             connect_at_start=ETHEREUM_NODES_TO_CONNECT_AT_START,
         )
+
+        # TODO: decide how do we want it in ChainManager:
+        # 1. A single argument/property (i.e. substrate_manager=substrate_manager)
+        # 2. An argument/property per substrate chain (as for now)
+        kusama_manager = SubstrateManager(
```

Comment:
Which option do you prefer, a constructor argument per chain or joining the supported substrate ones under a single "substrate" object that we pass to the ChainManager?